devops/jobs/UserRetirementArchiver.groovy

```diff
@@ -77,7 +77,7 @@ class UserRetirementArchiver {
             parameters {
                 stringParam(
                     'COOL_OFF_DAYS',
-                    '67',
+                    '74',
```

Comment:
Nit: Please document reasoning for change in the commit body

Reply:
Sure thing. Does the reasoning in the comment make sense to you?